quickwit/quickwit-ingest/src/ingest_v2/ingester.rs

```diff
@@ -218,14 +219,17 @@ impl Ingester {
         Ok(())
     }
 
-    async fn create_shard<'a>(
+    async fn init_primary_shard(
```

Comment:
Can you add a comment explain what `init_primary_shard` does? (create the queue on the leader and possibly the followed and creates the replication stream)

Reply:
Will do.